docusaurus.config.js

```diff
@@ -415,6 +411,10 @@ module.exports = {
             to: '/how-to-guides/new-user-guides/manage-clusters/rotate-encryption-key',
             from: '/how-to-guides/advanced-user-guides/manage-clusters/rotate-encryption-key'
           },
+          {
+            to: '/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates',
+            from: '/how-to-guides/advanced-user-guides/manage-clusters/manage-cluster-templates'
```

Comment:
```suggestion
            from: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates'
```

Suggestion made assuming the goal is the redirect from where this file was before it got moved/renamed as part of this PR. In any case, it doesn't look like the file was named/located at `/how-to-guides/advanced-user-guides/manage-clusters/manage-cluster-templates` at any point.